toto/util.py

```diff
@@ -1,31 +1,108 @@
-"""
-    TODO: markup
-"""
```

Comment:
Indeed, this is missing docstrings. Maybe I should have omitted the *everywhere* in the PR's title.